ui/collapsible-panel.reel/collapsible-panel.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @module ui/collapsible-panel.reel
+ * @requires montage/ui/component
+ */
+var Component = require("../component").Component,
+    PressComposer = require("../../composer/press-composer").PressComposer;
+
+/**
+ * @class CollapsiblePanel
+ * @extends Component
+ *
+ * A collapsible panel that can be expanded or collapsed with header and body
+ */
+exports.CollapsiblePanel = Component.specialize(/** @lends CollapsiblePanel# */ {
+    constructor: {
+        value: function CollapsiblePanel() {
+            this.super();
+            this._pressComposer = new PressComposer();
+        }
+    },
+
+    _pressComposer: {
+        value: null
+    },
+
+    /**
+     * Set this to true when clicking on header to collapse or expand a panel, only trgiiger animation at this stage.
+     * No animation when first time drawing
+     * @private
+     */
+    _isCollapsing: {
+        value: false
+    },
+
+    _isExpanded: {
+        value: null
+    },
```

Comment:
I don't think defining "_isExpanded" is necessary, and it would be simpler for the definition to be removed. How is it done in other classes that have public properties with setters than set a corresponding private property?